src/data/downloads.tsx

```diff
@@ -16,7 +16,8 @@ export enum OsType {
   Docker,
   Linux,
   MacOS,
-  Windows
+  Windows,
+  DotNet,
```

Comment:
```suggestion
  .NET,
```
dotnet branding guidelines for reference
https://github.com/dotnet/brand/blob/main/dotnet-brand-guidelines.pdf

Reply:
Internal enum should not care about branding that much as that is just like source code variable name

Reply:
Btw I don’t think you can use the dot in enum name at all

Reply:
Sorry wrong place, it should be below